### PR TITLE
#5 - WIP - HTTP Call to Hukamnama API + more

### PR DESCRIPTION
**Pull request by plug-n-play**
Sorry for the long PR; PR for #5 

1. Added HTTP Call to fetch Daily Hukamnama
2. Removed Hard-coding
3. Added hard-coded settings; now functional for Verse (Baani)
4. Updated to latest version of Next.js which renamed `static` folder to `public`; `public` name is not needed when specifying paths

### CODE REVIEW

**Comment by harpreetkhalsagtbit** [comment]
I have gone through this PR.
There are a lot of changes and its difficult to review. Also, I can see that most of the code has been taken from the old version.

My suggestion is to merge this into the `dev` branch. Then start refactoring it page by page.

I can see too much code in one single file. We definitely need to change this and instead of writing sub-component in the same file create a new file. For this, we also need to create a well-defined folder structure.
eg:

Component

         -> Home

             -> index.js (home)

          -> Common

              -> Header

                  -> index.js(header)

I am sure, we will be using Redux, **[Duck Pattern](https://medium.com/swlh/the-good-the-bad-of-react-redux-and-why-ducks-might-be-the-solution-1567d5bdc698)** is what is suggested for folder structure which in short suggests keeping all the related files for one particular component in the same folder which includes its `.js, .css, actions, reducers, tests` altogether in the same folder. We can have a discussion on it.

Start using `ReactTestingLibrarary` from the beginning.

Avoid Common mistakes while working with React. It's for all of us whether beginner or experienced.

Please follow the articles below:

[Common used React Patterns ](https://reactpatterns.com/)

[21 React perf tips - Things to do or not to do in react](https://medium.com/better-programming/https-medium-com-mayank-gupta-6-88-21-performance-optimizations-techniques-for-react-d15fa52c2349)

Rest we can discuss later let's just first focus on the setting up the base first instead of moving features quickly. Next.js can't help us if we are still doing the same mistakes. The more effectively we will use react the less complex our code would be. 

Don't worry we are all beginners here, we will learn and develop it in the best way possible.

**In short**
1. Let's merge this as it is.
2. Draft a plan for future tasks.
3. Setup basic architecture for the project.
4. Focus on small things and ship features with quality code even if it is just a Header Component.
5. Document code - As we are referring old code and picking whatever we can from the older version, we need to be careful as we are not aware of why or what that old code does exactly. We may need that in the future. We need to document as much as possible. What and why we changed?

Rest we can figure out later. Refactoring or Re-write is more difficult than starting a new project. As we are not supposed to commit the same mistakes again otherwise what's the use of doing that.

We are open to suggestions. Please bless us with your opinions as well.

**Comment by plug-n-play** [comment]
As suggested, merging this PR as is.

The current approach being taken was to have a replica of existing codebase to Nextjs, but without Redux and using ContaxtAPI for state management.

The mark-up was taken as is, initially inserted into `dangerouslySetHTML` so that the UI starts appearing on the screen, and then create components out of it.

Lets discuss what all can be done better.
